framework/base/CComponent.php

```diff
@@ -609,7 +609,14 @@ public function evaluateExpression($_expression_,$_data_=array())
 		if(is_string($_expression_))
 		{
 			extract($_data_);
-			return eval('return '.$_expression_.';');
+			try
+			{
+				return @eval('return ' . $_expression_ . ';');
```

Comment:
That changes behavior to be silent if PHP version is lower than 7. I don't think we should change it.